pkg/comp-functions/functions/vshnkeycloak/deploy.go

```diff
@@ -700,6 +701,40 @@ func newValues(ctx context.Context, svc *runtime.ServiceRuntime, comp *vshnv1.VS
 		"podSecurityContext": nil,
 	}
 
+	if comp.Spec.Parameters.Service.CustomEnvVariablesRef != nil {
+		envFrom, err := toYAML(comp.Spec.Parameters.Service.CustomEnvVariablesRef)
+		if err != nil {
+			return nil, err
+		}
+
+		values["extraEnvFrom"] = envFrom
+	}
```

Comment:
This should probably be removed as it's not really needed and will be overridden further down